src/consensus/raft/test/main.cpp

```diff
@@ -678,7 +700,9 @@ TEST_CASE("Exceed append entries limit")
 
   for (size_t i = 1; i <= num_big_entries; ++i)
   {
-    REQUIRE(r0.replicate({{i, data, true}}));
+    BatchType batch;
+    batch.emplace_back(i, data, true);
+    REQUIRE(r0.replicate(batch));
```

Comment:
because `replicate` is templated `T` doesn't get inferred if you do `r0.replicate({i, data, true})`, or `r0.replicate({i, {1, 2, 3}, true})` Any suggestions on how to work around it and avoid the boilerplate?

Reply:
`r0.replicate(BatchType{i, data, true})`?

Reply:
`r0.replicate(Batch{BatchTuple(i, data, true)})`